bigtop-packages/src/common/oozie/patch3-AD-151.diff

```diff
@@ -0,0 +1,341 @@
+ pom.xml                          | 61 ++++++++++++++++++++++++++++++--
+ server/pom.xml                   |  4 +++
+ sharelib/distcp/pom.xml          |  7 ++++
+ sharelib/hive/pom.xml            |  4 +++
+ sharelib/pig/pom.xml             |  4 +++
+ sharelib/spark/pom.xml           |  4 +++
+ zookeeper-security-tests/pom.xml | 12 +++++++
+ 9 files changed, 118 insertions(+), 5 deletions(-)
+
+diff --git a/core/pom.xml b/core/pom.xml
+index f582ed16a..849c1b38c 100644
+--- a/core/pom.xml
++++ b/core/pom.xml
+@@ -256,8 +256,9 @@
+         </dependency>
+ 
+         <dependency>
+-            <groupId>log4j</groupId>
+-            <artifactId>log4j</artifactId>
++            <groupId>ch.qos.reload4j</groupId>
++            <artifactId>reload4j</artifactId>
++            <version>${reload4j.version}</version>
+             <scope>compile</scope>
+         </dependency>
+ 
+@@ -265,6 +266,12 @@
+             <groupId>log4j</groupId>
+             <artifactId>apache-log4j-extras</artifactId>
+             <scope>compile</scope>
++            <exclusions>
++              <exclusion>
++                <groupId>log4j</groupId>
++                <artifactId>log4j</artifactId>
++              </exclusion>
++            </exclusions>
+         </dependency>
+ 
+         <dependency>
+@@ -490,6 +497,12 @@
+             <groupId>org.apache.hive</groupId>
+             <artifactId>hive-jdbc</artifactId>
+             <scope>compile</scope>
++            <exclusions>
++                <exclusion>
++                    <groupId>log4j</groupId>
++                    <artifactId>log4j</artifactId>
++                </exclusion>
++            </exclusions>
+         </dependency>
+         <dependency>
+             <groupId>net.jcip</groupId>
+@@ -522,6 +535,12 @@
+             <artifactId>hive-service</artifactId>
+             <version>${hive.version}</version>
+             <scope>test</scope>
++            <exclusions>
++                <exclusion>
++                    <groupId>log4j</groupId>
++                    <artifactId>log4j</artifactId>
++                </exclusion>
++            </exclusions>
+         </dependency>
+ 
+     </dependencies>
+diff --git a/examples/pom.xml b/examples/pom.xml
+index 4e10e63b6..8b09f4a77 100644
+--- a/examples/pom.xml
++++ b/examples/pom.xml
+@@ -152,6 +152,10 @@
+                     <groupId>org.slf4j</groupId>
+                     <artifactId>jcl-over-slf4j</artifactId>
+                 </exclusion>
++                <exclusion>
++                    <groupId>log4j</groupId>
++                    <artifactId>log4j</artifactId>
++                </exclusion>
+             </exclusions>
+         </dependency>
+     </dependencies>
+diff --git a/pom.xml b/pom.xml
+index 110daeb21..ba1b1ee93 100644
+--- a/pom.xml
++++ b/pom.xml
+@@ -117,6 +117,7 @@
+          <curator.version>2.5.0</curator.version>
+          <jackson.version>2.6.5</jackson.version>
+          <log4j.version>1.2.17</log4j.version>
```

Comment:
We can remove this property as it is not used anywhere, right?
` <log4j.version>1.2.17</log4j.version>`

Reply:
It's used at apache-log4j-extras